src/components/tag/tag.js

```diff
@@ -0,0 +1,30 @@
+import "./tag.scss";
+
+import classnames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+const Tag = ({ type, children, round }) => {
```

Comment:
This is an incorrect name for the component. Tag suits better for program identification. 
It should be Chips, i.e. https://material-ui.com/demos/chips/